dhctl/cmd/dhctl/commands/mirror/image/copy.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2023 Flant JSC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package image
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/containers/image/v5/copy"
```

Comment:
This is something we would like to avoid. Could you elaborate on why you decided to pick this library? 

1. In Deckhouse we already using the [go-containerregistry](https://github.com/google/go-containerregistry).
2. I had a talk with folks from werf, and they also said, that they only use the go-containeregistry.

Are there any specific reasons to use it?